create_amazon_bucket.md

```diff
@@ -0,0 +1,77 @@
+## Create a new amazon bucket
+
+### Bucket
+Use the naming convention `<product-name>-production`, `<product-name>-staging`
+
+#### Region
+We normally default to use `EU-Stockholm` but in theory this isn't as important, so if another region makes more sense choose a different one.
+
+#### Settings
+Under the section `Configure options` as a default we disable everything, it's only if the project require something specific such as versioning or server logging you should add it.
+
+The only thing that is recommended, is to add `default encryption` with AES-256. This should be used if the bucket is suppose to handle sensitive information(ID's, documentation that can be referenced back to the client etc.)
+
+#### Permissions
+Block all public access, unless the bucket is accessed by a service where it's not required to be logged in.
+It's fine to block all public access, you just have to create a token in your service that grants access to the data(Remember to set a time limit on the token)
+
+### IAM user
+You should never use the root access in your service, or the login that where used to create the bucket. There should be a IAM user for each bucket created, that way if one user is compromised, then it's only that specific bucket that is damaged.
+
+1. Goto Security, Identity & Compliance < IAM
+2. Then Access Management < Users
+3. Click add user.
+
+
+To make it easy to quickly connect the user to the bucket, when looking at the console, make the user name identical to the bucket name.
+Since this is a user only used through the service/application, the only `access type` it should have access to is `Programmatic access`
```

Comment:
```suggestion
In order to easily identify which user connects to which bucket, name them both the same.  Since this is a user only used through the service/application, the only `access type` it requires is `Programmatic access`,
```